common/src/main/java/com/wynntils/handlers/scoreboard/ScoreboardHandler.java

```diff
@@ -172,26 +172,38 @@ private boolean isScoreboardValid(List<ScoreboardLine> reconstructedScoreboard)
                 .map(ScoreboardLine::line)
                 .orElse(StyledText.EMPTY)
                 .equals(StyledText.fromString("À"))) {
-            return false;
+            // We did not find a new line at the start, so the scoreboard is invalid
+            return List.of();
         }
 
         // 4. Check for segment correctness
+        //    There are 2 error cases here:
+        //       - Fatal error: We find info that makes the current scoreboard invalid
+        //       - "Valid" error: We find an error, but it only makes the current scoreboard invalid, not the whole
```

Comment:
What does the "not the whole" mean, is this meant to be it only makes the current segment invalid, not the whole scoreboard?

Reply:
fixed the comment